app/src/main/kotlin/org/eurofurence/connavigator/ui/LoginActivity.kt

```diff
@@ -195,14 +195,26 @@ class LoginUi : AnkoComponent<LoginActivity> {
                 verticalLayout {
                     visibility = if (AuthPreferences.isLoggedIn()) View.VISIBLE else View.GONE
 
-                    textView("Username: ${AuthPreferences.username}")
-                    textView("Login is valid until: ${DateTime(AuthPreferences.tokenValidUntil).toLocalDateTime()}")
+                    textView("Username: ${AuthPreferences.username}") {
```

Comment:
Margin can be delared outside of lparams. Removing the indenting makes it a lot more readable